frontend/src/scenes/notebooks/NotebookPanel/notebookPopoverLogic.ts

```diff
@@ -1,122 +1,25 @@
-import { actions, kea, listeners, path, reducers, selectors } from 'kea'
+import { actions, kea, path, reducers } from 'kea'
 import { urlToAction } from 'kea-router'
 import { subscriptions } from 'kea-subscriptions'
 import posthog from 'posthog-js'
-import { HTMLProps, RefObject } from 'react'
 
-import { NotebookNodeResource, NotebookPopoverVisibility } from '~/types'
+import { NotebookPopoverVisibility } from '~/types'
 
-import { EditorFocusPosition } from '../Notebook/utils'
 import type { notebookPopoverLogicType } from './notebookPopoverLogicType'
 
 export const notebookPopoverLogic = kea<notebookPopoverLogicType>([
     path(['scenes', 'notebooks', 'Notebook', 'notebookPopoverLogic']),
     actions({
-        setFullScreen: (full: boolean) => ({ full }),
-        selectNotebook: (id: string, autofocus: EditorFocusPosition | undefined = undefined) => ({ id, autofocus }),
-        setElementRef: (element: RefObject<HTMLElement>) => ({ element }),
         setPopoverVisibility: (visibility: NotebookPopoverVisibility) => ({ visibility }),
-        startDropMode: true,
-        endDropMode: true,
-        setDropDistance: (distance: number) => ({ distance }),
-        setDroppedResource: (resource: NotebookNodeResource | string | null) => ({ resource }),
     }),
 
     reducers(() => ({
-        selectedNotebook: [
-            'scratchpad',
-            { persist: true },
-            {
-                selectNotebook: (_, { id }) => id,
-            },
-        ],
         popoverVisibility: [
```

Comment:
This can also go.

The connection in `notebookPanelLogic` is not needed anymore

https://github.com/PostHog/posthog/blob/72064c8b42f8cf6ebaf50a2576ace151a0442f95/frontend/src/scenes/notebooks/NotebookPanel/notebookPanelLogic.ts#L73-L74

And the only types needed are `visible` and `hidden`. We can probably make that a boolean but I can follow up there to keep this PR more contained